srfi-179.html

```diff
@@ -35,7 +35,8 @@ <h2>Status</h2>
         <ul>
           <li>2020-10-08 (Fix documentation for <code>interval-subset?</code>.)</li>
           <li>2021-02-24 (Correct the note comparing  <code>array-curry</code> and <code>array-tile</code>.)</li>
-          <li>2021-05-02 (Correct documentation of <code>interval-cartesian-product</code>; document initial values for <code>(specialized-array-default-safe?)</code> and <code>(specialized-array-default-mutable?)</code>.</li></ul></li>
+          <li>2021-05-02 (Correct documentation of <code>interval-cartesian-product</code>; document initial values for <code>(specialized-array-default-safe?)</code> and <code>(specialized-array-default-mutable?)</code>.)</li>
+          <li>2021-12-22 (Use <code>vector-concatenate</code> in place of <code>append-vectors</code> in reference implementation.)</li></ul></li>
```

Comment:
There's no need to note a change to the sample implementation in the document.  As long as the API is unchanged, only the implementation need change.